### ISSUE TITLE
Flightpath thru animated fractal

### ISSUE THREAD

**Issue by mclarekin** [buddhi1980/mandelbulber2#218]
With openCL we may be able to setup an animated fractal, link it to flight animation, and creates flight path by navigating thru  the animation. Avoiding points that are animating towards  you and your flight path ahead, and navigating towards parts that appear to be morphing in an interesting way.

If you know what I mean :)

**Comment by mancoast** [comment]
This requires a dynamic collision avoidance system for 3D Fractal Solid Bodies that morph to audio transform.

**Comment by buddhi1980** [comment]
I think @mclarekin has another idea. First to prepare fractal which is changing shape. Then to use this already morphed fractal in Record Flight. It could be used in connection with AnimBySound (but not only)

**Comment by mclarekin** [comment]
Yes Krzysztof, that was what I was thinking. Flying through an animated 3D
world, to be evaluated after implementation of openCL.



On Wed, Mar 15, 2017 at 7:08 AM, Krzysztof Marczak <notifications@github.com

**Comment by mancoast** [comment]
Bump to opencl, 2.12

**Comment by mclarekin** [comment]
or 2.13. Getting openCL working is 2.12 this can be done afterwards.

**Comment by mancoast** [comment]
@buddhi1980 @mclarekin I think this issue is very intriguing with the GPU. Is it possible to record a flight path through dynamic fractal objects? 
@mclarekin As I understand your meaning: Lets say that we have some parameters of the fractal object morphing and changing in the animation engine. Now that the object is changing, we want to fly around and record a new flight path within this morphing and changing fractal object.
Is this on the right track?

I want to accomplish this, but also extend it into the animation by sound subsystem.  Imagine that the fractal object is morphing based on the fft of the audio signal.  Can we record the flight path through this type of dynamic fractal object?

What do you think?

**Comment by buddhi1980** [comment]
We can do such a recording mode. It is possible. But coding is needed. Who will do it?

**Comment by mclarekin** [comment]
Nice to have, but I am thinking there are  other animation and audio
animation enhancements to  be implemented before looking at this. I am
intending to start working  on animation in V2.19
